pkg/apis/config/store.go

```diff
@@ -56,13 +58,16 @@ func FromContextOrDefaults(ctx context.Context) *Config {
 	artifactPVC, _ := NewArtifactPVCFromMap(map[string]string{})
 	metrics, _ := newMetricsFromMap(map[string]string{})
 	trustedresources, _ := NewTrustedResourcesConfigFromMap(map[string]string{})
+	spireconfig, _ := NewSpireConfigFromMap(map[string]string{})
```

Comment:
do you know why the returned errors are being ignored here from this function call and the ones above? it seems like we should probably handle them?

Reply:
+1 to handling the errors.
This is more a design question. If there is an error in one of the configs then we could return nil from this function stating that the function failed. This might not work well as and error in config should not be reason for stalling the initialization.
These should not be propagated upwards and handling these could just be logging the error and continuing.

Reply:
would it make sense to log something in the case of an error at least? (or maybe that is already happening?)